.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+name: Backend CI
```

There is no other CI here, e.g. frontend?
```suggestion
name: CI
```